packages/x-components/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+const { xTailwindPlugin } = require('@empathyco/x-tailwindcss');
+
+module.exports = {
+  content: [
+    './index.html',
+    './src/components/**/*.vue',
+    './src/views/**/*.vue',
+    './src/x-modules/**/components/**/*.vue'
+  ],
+  prefix: 'x-',
```

Comment:
The issue with the prefix is that tailwind does not eliminate all unused CSS when we use the same `x` prefix that we currently have for the old design system. It includes some classes that have the same name but different use cases (x-filter for example), so that's why I would encourage using a different prefix.

Reply:
As agreed with @tajespasarela, we will keep the `x` prefix. The idea of using `xds` was to ease the migration process, but it would end up being more complicated, as we would have to keep both classes alive while the migration is ongoing, and once the first full version of the XDS builder is finished remove all `xds` classes. 

To do things step by step and avoid conflicting classes, we are also going to [disable all tailwind core plugins](https://tailwindcss.com/docs/theme#disabling-an-entire-core-plugin), but one or two to ensure that tailwind keeps working because they are 112 plugins, here is the config below:

```
{
    "screens": false,
    "colors": false,
    "columns": false,
    "spacing": false,
    "animation": false,
    "aspectRatio": false,
    "backdropBlur": false,
    "backdropBrightness": false,
    "backdropContrast": false,
    "backdropGrayscale": false,
    "backdropHueRotate": false,
    "backdropInvert": false,
    "backdropOpacity": false,
    "backdropSaturate": false,
    "backdropSepia": false,
    "backgroundColor": false,
    "backgroundImage": false,
    "backgroundOpacity": false,
    "backgroundPosition": false,
    "backgroundSize": false,
    "blur": false,
    "brightness": false,
    "borderColor": false,
    "borderOpacity": false,
    "borderRadius": false,
    "borderSpacing": false,
    "borderWidth": false,
    "boxShadow": false,
    "boxShadowColor": false,
    "caretColor": false,
    "accentColor": false,
    "contrast": false,
    "container": false,
    "content": false,
    "cursor": false,
    "divideColor": false,
    "divideOpacity": false,
    "divideWidth": false,
    "dropShadow": false,
    "fill": false,
    "grayscale": false,
    "hueRotate": false,
    "invert": false,
    "flex": false,
    "flexBasis": false,
    "flexGrow": false,
    "flexShrink": false,
    "fontFamily": false,
    "fontSize": false,
    "fontWeight": false,
    "gap": false,
    "gradientColorStops": false,
    "gridAutoColumns": false,
    "gridAutoRows": false,
    "gridColumn": false,
    "gridColumnEnd": false,
    "gridColumnStart": false,
    "gridRow": false,
    "gridRowStart": false,
    "gridRowEnd": false,
    "gridTemplateColumns": false,
    "gridTemplateRows": false,
    "height": false,
    "inset": false,
    "keyframes": false,
    "letterSpacing": false,
    "lineHeight": false,
    "listStyleType": false,
    "margin": false,
    "maxHeight": false,
    "maxWidth": false,
    "minHeight": false,
    "minWidth": false,
    "objectPosition": false,
    "opacity": false,
    "order": false,
    "padding": false,
    "placeholderColor": false,
    "placeholderOpacity": false,
    "outlineColor": false,
    "outlineOffset": false,
    "outlineWidth": false,
    "ringColor": false,
    "ringOffsetColor": false,
    "ringOffsetWidth": false,
    "ringOpacity": false,
    "ringWidth": false,
    "rotate": false,
    "saturate": false,
    "scale": false,
    "scrollMargin": false,
    "scrollPadding": false,
    "sepia": false,
    "skew": false,
    "space": false,
    "stroke": false,
    "strokeWidth": false,
    "textColor": false,
    "textDecorationColor": false,
    "textDecorationThickness": false,
    "textUnderlineOffset": false,
    "textIndent": false,
    "textOpacity": false,
    "transformOrigin": false,
    "transitionDelay": false,
    "transitionDuration": false,
    "transitionProperty": false,
    "transitionTimingFunction": false,
    "translate": false,
    "width": false,
    "willChange": false,
    "zIndex": false
}
```

Then I'll create another task to create a system to allow dynamic css classes in the components. It doesn't make sense to allow tailwind to set a custom class prefix, while x-components don't support it. This system should also allow to add custom class names to every node in the DOM.